extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -163,13 +215,17 @@ export class RemoteQueriesManager {
     };
   }
 
+  public async openResults(query: RemoteQuery, queryResult: RemoteQueryResult) {
```

Comment:
The public functions of the `RemoteQueriesManager` are `runRemoteQuery`, `monitorRemoteQuery`, etc. so perhaps a better name for this function would be `openRemoteQuery` or `showRemoteQuery`.